src/components/GetInTouch.astro

```diff
@@ -86,8 +86,7 @@
 		<span>
 			Press inquiries may be directed via email to Andrew at
 			<a href="mailto:andrew@the-collab-lab.codes">
-				andrew@the-collab-lab.codes
-			</a>.
+				andrew@the-collab-lab.codes</a>.
```

Comment:
I think, technically, this change isn't necessary. Other links on the page have whitespace between the link text and the closing anchor tag and it doesn't cause any issues.